next.config.js

```diff
@@ -3,14 +3,15 @@ const { i18n } = require('./next-i18next.config')
 const isProd = process.env.NODE_ENV === 'production'
 
 module.exports = {
-  // The Claim Tracker application must live at /claimstatus because IDM Webgate 
+  // The Claim Tracker application must live at /claimstatus because IDM Webgate
   // expects the "context root" to be /claimstatus, and all other pages must be sub-paths.
   basePath: '/claimstatus',
   assetPrefix: isProd ? '/claimstatus' : '',
   i18n,
   webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
-    config.node = {
-      fs: 'empty'
+    config.resolve.fallback = {
```

Comment:
This is the standard recommendation when migrating from webpack 4 to 5, e.g. see https://dev.to/marcinwosinek/how-to-add-resolve-fallback-to-webpack-5-in-nextjs-10-i6j